src/main/java/org/commonjava/util/sidecar/services/ProxyService.java

```diff
@@ -102,6 +106,12 @@ void handleConfigChange( String message )
         logger.debug( "Handle event {}, refresh timeout: {}", EVENT_PROXY_CONFIG_CHANGE, timeout );
     }
 
+    public static boolean doProxy( final String path )
```

Comment:
Seems static method in a CDI bean again. Maybe non-static or helper class?

Reply:
@ligangty to amend all the static methods